docs/usage.rst

```diff
@@ -5,12 +5,16 @@
 Usage
 =====
 
-|project_name| consists of a number of commands. Each command accepts
-a number of arguments and options. The following sections will describe each
-command in detail.
+``borg-import`` consists of a number of commands, one for each
+backup system supported. Each accepts a number of arguments and
+options. The following sections will describe each in detail.
 
 General
 -------
 
-...
+.. _rsnapshot:
 
+borg-import rsnapshot
+---------------------
+
+.. generate-usage:: rsnapshot
```

Comment:
Since it would pull the build_usage from setup.py I'd make the build_man part a separate script altogether, placed in docs/, and executed by `make man` (which would currently generate the not-nearly-as-nice Sphinx man page).

With these two one could also pull build_api from setup.py as well.